tests/test_extensions_build.py

```diff
@@ -10,13 +10,13 @@
 
 @pytest.mark.parametrize("venv_type, package_type,install_type",
                          [('venv', 'develop', 'GPU')])
-def test_force_cuda_build(create_tmp_venv_with_nncf, install_type, tmp_path, package_type):
+def test_force_cuda_build(tmp_venv_with_nncf, install_type, tmp_path, package_type):
     '''
     Check that CUDA Extensions weren't initially built and \
     then with TORCH_CUDA_ARCH_LIST were forced to be built
     '''
 
-    venv_path = tmp_path / 'venv'
+    venv_path = tmp_venv_with_nncf
```

Comment:
Anyway we have `tmp_path` as input. It is better to use only one way `tmp_path` or return value of `tmp_venv_with_nncf`

Reply:
The tmp path with venv and the tmp path with the checks scripts should not necessarily be the same. In fact, it is even beneficial this way - less chance of common PYTHONPATH influence.